src/components/attendee/VideoComms/types.tsx

```diff
@@ -0,0 +1,94 @@
+import Twilio from "twilio-video";
+
+export enum VideoCommsStatus {
+  Disconnected = "DISCONNECTED",
+  Connecting = "CONNECTING",
+  Connected = "CONNECTED",
+  Errored = "ERRORED",
+}
+
+export interface VideoCommsProviderProps {
+  userId: string;
+  children: React.ReactNode;
+}
+
+interface Track {
+  kind: "video" | "audio";
+  id: string;
```

Comment:
Maybe we add one more line in `./src/types/id.ts` ?

```suggestion
  id: TrackId;
```